src/SonarScanner.MSBuild.Common/StringExtensions.cs

```diff
@@ -40,5 +40,11 @@ public static class StringExtensions
         public static string ReplaceCaseInsensitive(this string input, string oldValue, string newValue) =>
```

Comment:
Outdated, because the extension method was moved as suggested in https://github.com/SonarSource/sonar-scanner-msbuild/pull/2018#discussion_r1668340483